app/src/main/res/values/styles.xml

```diff
@@ -8,4 +8,54 @@
         <item name="colorAccent">@color/colorAccent</item>
     </style>
 
+    <style name="GridItemBkgStyle">
+        <item name="android:layout_width">wrap_content</item>
+        <item name="android:layout_height">wrap_content</item>
+        <item name="android:background">@color/light_gray</item>
+        <item name="android:gravity">center</item>
+        <item name="android:orientation">vertical</item>
+        <item name="android:padding">5dp</item>
+    </style>
+
+    <style name="TextViewStyle">
+        <item name="android:layout_width">match_parent</item>
+        <item name="android:layout_height">wrap_content</item>
+        <item name="android:layout_above">@+id/iv_movie_poster</item>
+        <item name="android:gravity">center_horizontal</item>
+        <item name="android:maxLines">2</item>
+        <item name="android:layout_marginBottom">2dp</item>
+        <item name="android:textColor">@android:color/black</item>
+        <item name="android:textSize">20sp</item>
+
+    </style>
+
+    <style name="MovieImageItemStyle">
+        <item name="android:layout_width">185dp</item>
+        <item name="android:cropToPadding">true</item>
+        <item name="android:layout_alignParentBottom">true</item>
+        <item name="android:layout_height">200dp</item>
+    </style>
+
+    <style name="MovieInfoImageStyle">
+        <item name="android:layout_width">240dp</item>
+        <item name="android:cropToPadding">true</item>
+        <item name="android:layout_alignParentStart">true</item>
+        <item name="android:layout_alignParentLeft">true</item>
+        <item name="android:layout_height">300dp</item>
+    </style>
+
+    <style name="InfoTextViewStyle">
+        <item name="android:layout_width">wrap_content</item>
```

Comment:
Width and height are not needed within style if they are wrap_content or match_parent.
Style for width and height is used for special sizes...